app/services/analytics_events.rb

```diff
@@ -1387,6 +1387,68 @@ def personal_key_alert_about_sign_in(success:, errors:, **extra)
     )
   end
 
+  # param [Boolean] success
+  # param [Hash] errors
+  # param [Hash] error_details
+  # param [String] delivery_preference
+  # param [Integer] phone_configuration_id
+  # param [Boolean] make_default_number
+  # User has submitted a change in phone number
+  def phone_change_submitted(
+    success:,
+    errors:,
+    delivery_preference:,
+    phone_configuration_id:,
+    make_default_number:,
+    **extra
+  )
+    track_event(
+      'Phone Number Change: Form submitted',
+      success: success,
+      errors: errors,
+      delivery_preference: delivery_preference,
+      phone_configuration_id: phone_configuration_id,
+      make_default_number: make_default_number,
+      **extra,
+    )
+  end
+
+  # User has viewed the change in phone number page
```

Comment:
maybe rephrase this?
```suggestion
  # User has viewed the page to change their phone number
```